src/label/creator.py

```diff
@@ -16,7 +16,12 @@
 QR_CODE_ERROR_CORRECTION = qrcode.constants.ERROR_CORRECT_M
 QR_CODE_DESCRIPTION_MAX_LENGTH = 100
 
-IMG_WIDTH = 696  # From brother_ql for 62 mm labels
+# For Brother QL-810W with 62 mm wide labels
+PRINTER_HEIGHT_MARGIN_MM = 3
+PRINTER_PIXELS_PER_MM = 300 / 25.4
+PRINTER_LABEL_PRINTABLE_WIDTH = 58
+
+IMG_WIDTH = math.floor(PRINTER_PIXELS_PER_MM * PRINTER_LABEL_PRINTABLE_WIDTH)
 IMG_HEIGHT = math.floor((58 + 20) / 25.4 * 300)
```

Comment:
```suggestion
IMG_HEIGHT = math.floor((58 + 20) * PRINTER_PIXELS_PER_MM)
```

Reply:
Not the same thing, this is connected to the first labels developed, didn't dare too touch them without going thru what the effects would be.